{{cookiecutter.project_slug}}/templates/_base.html

```diff
@@ -1,12 +1,12 @@
 {% extends "base_templates/_base.html" %}
 {% block head %}
 {{super()}}
-<link rel="stylesheet" href="/static/style.css">
+<link rel="stylesheet" href="./static/style.css">
```

Comment:
As mentioned in https://github.com/render-engine/render-engine/issues/608

We need to provide an absolute path to prevent subdomain errors.